channeldb/invoices.go

```diff
@@ -174,12 +191,6 @@ type Invoice struct {
 	// or any other message which fits within the size constraints.
 	Memo []byte
 
-	// Receipt is an optional field dedicated for storing a
-	// cryptographically binding receipt of payment.
-	//
-	// TODO(roasbeef): document scheme.
-	Receipt []byte
-
```

Comment:
Also not removed in first commit?

Reply:
we could remove it sooner and instead write a blank receipt, but it is done here because here this is the commit that actually modifies the on-disk format. if we do it sooner then the existing serialization code can't reference any (in theory) existing receipts, and means that there would side effects if an invoice is decoded and deserialized before running the migration.

Reply:
ok